libbeat/processors/add_host_metadata/add_host_metadata.go

```diff
@@ -0,0 +1,50 @@
+package add_host_metadata
+
+import (
+	sysinfo "github.com/elastic/go-sysinfo"
```

Comment:
@andrewkroh Did you have to add a `-` to the lib name? :-D

Reply:
With Go you don't need to add a package alias, just import plain old "github.com/elastic/go-sysinfo" and use `sysinfo.XYZ` in the code.

Reply:
It was mainly my IDE that complained :-)